docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -1,65 +1,124 @@
 = Using the Data Migration Tool
-:description: This guide explains how to migrate data from Hazelcast 4.x or 5.x clusters to newer 5.x clusters.
+:description: You can use the Data Migration Tool (DMT) to migrate your data from version 4.x or 5.x Open Source and Enterprise Edition Hazelcast clusters when upgrading to 5.3.x versions of Enterprise Edition, or moving to the latest {hazelcast-cloud} release.
```

Comment:
`to 5.3.x or newer versions` will future proof this doc.

Reply:
Updated